### PR TITLE
Use pbr module and semantic versioning

### PR DESCRIPTION
**Pull request by tareqalayan**

### CODE REVIEW

**Comment by tareqalayan** [comment]
https://github.com/eanxgeek/pytest_marker_bugzilla/pull/19

@eanxgeek @lukas-bednar please take a look

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+pytest
+python-bugzilla
```
Comment (lukas-bednar) [inline requirements.txt]: python-bugzilla >=0.6.2

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+pytest
```
Comment (lukas-bednar) [inline requirements.txt]: pytest >= 2.2.4

File: setup.cfg [hunk 3]
```diff
@@ -0,0 +1,29 @@
+[metadata]
+name = pytest-marker-bugzilla
+author = Eric Sammons
+author-email = elsammons@gmail.com
+summary = py.test bugzilla integration plugin, using markers
+description-file = README.md
+home-page = https://github.com/eanxgeek/pytest_marker_bugzilla.git
+license = GPL
```
Comment (lukas-bednar) [inline setup.cfg]: please be consistent, on the line 12 there is GPLv2

File: setup.py [hunk 4]
```diff
@@ -16,34 +17,12 @@ def run_tests(self):
         sys.path.insert(0, 'lib')
         pytest.main(self.test_args)
 
+os.environ['SKIP_GENERATE_AUTHORS'] = '1'
```
Comment (lukas-bednar) [inline setup.py]: why do you skip it ?
The AUTHORS file is not managed in this repo, so you can let pbr to include it ...

File: setup.cfg [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+classifier =
+    Development Status :: 3 - Alpha
+    Topic :: Utilities
+    License :: OSI Approved :: GNU General Public License 2 (GPL2)
```
Comment (lukas-bednar) [inline setup.cfg]: this is not valid classifier
`License :: OSI Approved :: GNU General Public License v2 (GPLv2)`

source: https://pypi.python.org/pypi?%3Aaction=list_classifiers

File: setup.cfg [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    Programming Language :: Python
+keywords =
+    pytest
+    bugzilla
+    plugin
+[files]
+packages =
+    pytest_marker_bugzilla
+[bdist_rpm]
+requires =
```
Comment (lukas-bednar) [inline setup.cfg]: missing pytest

File: setup.cfg [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+    python-bugzilla
```
Comment (lukas-bednar) [inline setup.cfg]: add version too

File: .gitignore [hunk 8]
```diff
@@ -1,7 +1,7 @@
 *.pyc
-*.cfg
 *.swp
 *.egg-info
 build
 dist
 MANIFEST
+ChangeLog
```
Comment (lukas-bednar) [inline .gitignore]: you probably will need add AUTHORS as well

**Comment by lukas-bednar** [comment]
There is some problem, the tests are failing https://travis-ci.org/rhevm-qe-automation/pytest-marker-bugzilla/builds/127824547

**Comment by eanxgeek** [comment]
Can you please create a new pull request based on master, new master branch should, now, include pull-request #18.

**Comment by lukas-bednar** [comment]
Tareq please address my comments in new PR.

File: setup.cfg (hunk 5 above)
Comment (tareqalayan) [inline setup.cfg]: this is listed in the URL you provided

Comment (lukas-bednar) [inline setup.cfg]:
```
"License :: OSI Approved :: GNU General Public License 2 (GPL2)" != "License :: OSI Approved :: GNU General Public License v2 (GPLv2)"
```
missing 'v' infront of '2', the PYPI will complain about it if you publish package.

**Pull request merged**